controllers/AuthController.js

```diff
@@ -29,72 +30,22 @@ export default class AuthController {
    *
    * @param {object} req express request object
    * @param {object} res express response object
+   * @param {function} next
    *
    * @returns {object} returns user data
    *
    * @memberof AuthController
    */
-  static async signUp(req, res) {
-    const {
-      firstName, lastName, email, password, confirmPassword
-    } = req.body;
-    const genericWordsArray = [firstName, lastName, 'Password', 'password', 123];
-    const genericWord = genericWordsArray.find(word => password.includes(word));
-    if (genericWord) {
-      return res.status(400).send({
-        errors: {
-          password: 'Do not use a common word as the password',
-        }
-      });
-    }
-
-    const foundUser = await User.findOne({ where: { email } });
-    if (foundUser) {
-      return res.status(409).send({
-        message: 'This User already exist',
-      });
-    }
-
-    if (password !== confirmPassword) {
-      return res.status(400).send({
-        message: "Password doesn't match, Please check you are entering the right thing!",
-      });
-    }
-    const hashedPassword = Helper.hashPassword(password);
-    const verificationToken = Helper.hashUserData(email);
-    const user = {
-      firstName,
-      lastName,
-      email,
-      isVerified: false,
-      verificationToken,
-      password: hashedPassword,
-      type: 'user',
-    };
-
-    const registeredUser = await User.create(user);
-    const token = Helper.createToken({
-      id: registeredUser.id,
-      firstName: registeredUser.firstName,
-      lastName: registeredUser.lastName,
-      isVerified: registeredUser.isVerified,
-      email,
-      type: registeredUser.type,
-    });
-    // This line sends the registered user an email
-    /* istanbul ignore next-line */
-    if (process.env.NODE_ENV !== 'test') {
-      EmailVerification.sendVerificationEmail(
-        req, registeredUser.email, registeredUser.firstName,
-        verificationToken
-      );
+  static async signUp(req, res, next) {
+    try {
+      const {
+        firstName, lastName, email, password, confirmPassword
+      } = req.body;
+      // const type = 'user';
```

Comment:
Hi, please, why is this line commented out? If it's not relevant anymore, wouldn't it make more sense to just delete it?

Reply:
it was an over sight thank you